job_bundles/README.md

```diff
@@ -0,0 +1,315 @@
+# AWS Deadline Cloud Job Bundles
+
+Job Bundles are one of the tools that you can use to define your Jobs for AWS Deadline Cloud. They encapsulate
+an [Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) into a directory
+with additional information such the files and directories that your Jobs need for
+Deadline Cloud's [Job Attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html)
+feature. The [Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud) provides ways for you to use a Job Bundle
+to submit Jobs to your Deadline Cloud Queues to be run.
+
+We recommend developing your own Job Bundle(s), either by-hand or programatically, to benefit from their
+integration with Job Attachments and an intuitive graphical Job submission interface. For example, after 
```

Comment:
This will be more clear without the job attachments reference. When you write a job bundle, you can structure it to work with both job attachments and shared file systems, and someone reading this might interpret it to mean that job bundles are for job attachments.